code/ui/components/src/new/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { forwardRef } from 'react';
+import { styled } from '@storybook/theming';
+import TextareaAutoResize from 'react-textarea-autosize';
+
+interface TextareaProps {
```

Comment:
Consider adding PropTypes for better type checking and to make the code more self-documenting. [medium]

Reply:
No again.